src/h5web/visualizations/heatmap/models.ts

```diff
@@ -3,10 +3,6 @@ import { INTERPOLATORS } from './interpolators';
 
 export type Dims = [number, number];
 
-export type Domain = [number, number];
-
-export type AxisOffsets = [number, number];
-
 export type D3Interpolator = (t: number) => string;
 
 export type ColorMap = keyof typeof INTERPOLATORS;
```

Comment:
I felt that models and utils related to colormap and color gradients should stay in `heatmap`